src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -502,87 +518,121 @@ private void testEditInstructorAction() throws Exception {
-        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, 0, 1);
-        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, 0, 0));
-        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, 0, 1));
-        
-        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, 0, 3);
-        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, 0, 2));
-        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, 0, 3));
-        
-        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, 0, 6);
-        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, 0, 5));
-        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, 0, 6));
+        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_GIVE_STUDENT_COMMENTS);
+        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_VIEW_STUDENTS));
+        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_GIVE_STUDENT_COMMENTS));
+        
+        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_MODIFY_OTHERS_COMMENTS);
+        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_VIEW_OTHERS_COMMENTS));
+        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_MODIFY_OTHERS_COMMENTS));
+        
+        courseEditPage.clickSectionLevelPrivilegeLink(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_MODIFY_RESPONSES_IN_SESSION);
+        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_VIEW_RESPONSES_IN_SESSION));
+        assertTrue(courseEditPage.isSectionLevelPrivilegeLinkClicked(editInstructorIndex, sectionToCheck,
+                InstructorCourseEditPage.SECTION_MODIFY_RESPONSES_IN_SESSION));
         
         // session level
-        courseEditPage.clickSessionLevelPrivilegeLink(editInstructorIndex, 0, 0, 2);
-        assertTrue(courseEditPage.isSessionLevelPrivilegeLinkClicked(editInstructorIndex, 0, 0, 1));
-        assertTrue(courseEditPage.isSessionLevelPrivilegeLinkClicked(editInstructorIndex, 0, 0, 2));
+        courseEditPage.clickSessionLevelPrivilegeLink(editInstructorIndex, sectionToCheck, sessionToCheck,
+                InstructorCourseEditPage.SESSION_MODIFY_RESPONSES);
```

Comment:
May be this is easier on the eye? i.e. the end of 2nd line occurs right of the first line (because the 2nd line contains the text that is supposed to be on the right of the first line). The idea is the 2-line version stays as close to the 1-line version so that the disruption caused by the line break is minimal.
```java
courseEditPage.clickSessionLevelPrivilegeLink(editInstructorIndex, 
        sectionToCheck, sessionToCheck, InstructorCourseEditPage.SESSION_MODIFY_RESPONSES);